### PR TITLE
changes from facetview feature_5959 branch

### PR DESCRIPTION
**Pull request by delsborg**
See https://github.com/cu-boulder/facetedsearch/pull/14 for details.
This change here just deploys the actual people.ftl page modified in above PR to Experts

### CODE REVIEW

File: webapp/src/main/webapp/themes/cu-boulder/templates/people.ftl [hunk 1]
```diff
@@ -19,49 +75,91 @@
...
     <link rel="stylesheet" href="/themes/cu-boulder/browsers.css">
+    <!-- Add Font Awesome icon library -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
     <!-- <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.7.5/leaflet.css" /> -->
 
     <script id="person-template" type="text/x-handlebars-template">
         <tr>
             <td>
+                </div>
```
Comment (aviggio) [inline webapp/src/main/webapp/themes/cu-boulder/templates/people.ftl]: Remove this extra closing </div>

File: webapp/src/main/webapp/themes/cu-boulder/templates/people.ftl [hunk 2]
```diff
@@ -182,16 +344,126 @@
...
+    </div>
+  </div>
 <div class="facet-view-simple">
-<div class="help"> <h3> PEOPLE SEARCH </h3> Use the People Search bar directly below or the expandable Filters at left to explore Boulder faculty data. People Search allows for wildcard * or exact search with " " double quotations. The research filter is only searching the Research Keywords, not the free-text keywords or research overview. To search VIVO without the filters in order to include all research fields, use the site search bar in the VIVO page header. Filters default to 'and' logic. Toggle with the 'or' button if desired.</div>
+</div>
 </body>
 </html>
+
```
Comment (aviggio) [inline webapp/src/main/webapp/themes/cu-boulder/templates/people.ftl:394]: Confirm if this extra code is correct... looks like extra /body and /html codes.

Comment (delsborg) [reply webapp/src/main/webapp/themes/cu-boulder/templates/people.ftl:394]: cleaned up the extra /body and /html along with the orphan </div>
Verified the code looks clean in chrome developer

**Comment by delsborg** [comment]
Requested changes have been made.
Additional fixes for Internet Explorer have been incorporated.
Merging now and minor fixes can be addressed later.

**Pull request merged**